source/clear-linux/tutorials/smb/smb-desktop.rst

```diff
@@ -0,0 +1,79 @@
+.. _smb-desktop:
+
+Connect to Windows\* shared location from |CL-ATTR| desktop
+###########################################################
+
+This tutorial explains how to access an existing shared location on
+Windows\* via Server Message Block (SMB) /Common Internet File System (CIFS)
+from the |CL| GNOME desktop.  CIFS filesystem is generally used to mount
+SMB/CIFS shares on Linux distributions. Though not really part of Samba,
+these tools were originally part of the Samba package.
```

Comment:
@phmccarty, or @mdhorn . Can you please verify that the package `storage-utils` is installed automatically with the live-desktop-beta image? I need to know this in case there were a prereq for implementing a SMB/CIFS share using Nautilus/File manager. I just need to cross this out as a possibility.  

FYI: I tried to uninstall `storage-utils` to test whether this would prevent setting up a share, but CL wouldn't allow me to do so. It showed error message of: 
-# * is-required-by 
[...]
* user-basic
* os-clr-on-clr
* storage-utils-dev
  |-- os-clr-on-clr